### ISSUE TITLE
How to fine tune Lora without images……

### ISSUE THREAD

**Issue by xiaxiaxiatengxi** [haotian-liu/LLaVA#1585]
### Describe the issue

Issue:
How to fine tune Lora without images……
I want to finetune a Lora without images and use GPTShare format 

Data Format: Share GPT and Multiple rounds conversations
{
        "conversations": [
            {
                "from": "human",
                "rewards": 0,
                "loss": null,
                "value": "WebShop [SEP] Instruction: [SEP] Find me space saving, easy clean baskets and storage for living room with color: grey |  beige, and price lower than 40.00 dollars [SEP] Search.The next action is ?"
            },
            {
                "from": "gpt",
                "rewards": 0.0,
                "loss": true,
                "value": "{\"Action\":\"search[space saving easy clean baskets storage living room grey beige under $40]\"}"
            },
            {
                "from": "human",
                "rewards": 0.0,
                "loss": true,
                "value": "WebShop [SEP] Instruction: [SEP] Find me space saving, easy clean baskets and storage for living room with color: grey |  beige, and price lower than 40.00 dollars [SEP] SearchInstruction: [SEP] Find me space saving, easy clean baskets and storage for living room with color: grey |  beige, and price lower than 40.00 dollars [SEP] Back to Search [SEP] Page 1 (Total results: 50) [SEP] Next > [SEP] B09JC84P1M [SEP] RUIXFLR Round End Table with Fabric Storage Basket, Marble Look Tray Side Table, Chic 2 Tier Coffee Table for Living Room Bedroom Office [SEP] $121.99 [SEP] B07FDJYBVW [SEP] Seacrest White Kitchen Cart by Linon [SEP] $331.77 [SEP] B09B7G7P58 [SEP] 3-Piece Dining Table Set, Farmhouse Counter Height Dining Table Set, Wood Dining Table Set with Drop Leaf Table/2 Cross Back Padded Chairs/One Shelf [SEP] $319.0 [SEP] B089B1K9LJ [SEP] Bxwjg Metal Filing Cabinet, 5-Layer File Storage Cabinet, with Lock and Index Label, Aluminum Alloy, for Business [SEP] $146.52 [SEP] B07SXRVKZ2 [SEP] Fab Habitat Seagrass Storage Basket Set - Wicker Pattern Baskets, Strong Handles - Organizer for Blankets, Towels, Pillows, Toys, Laundry, Baby, Kids, Home D\u00e9cor - Harlem - XL [SEP] $69.0 [SEP] B08R5Z7S2K [SEP] Estink File Cabinet, Grey Metal Moveable Hanging File Cabinet Nightstand Storage Cabinets with 5 Drawers Storage Filing Office Home Use,28 x 41 x 68.5 cm [SEP] $100.0 [SEP] B06XDG8XFX [SEP] CMA Stockroom Kids Wood 5-Pocket Sling Bookshelf Book Display Rack Color Walnut [SEP] $44.32 [SEP] B07FKGQKZ1 [SEP] Home Dynamix Lyndhurst Rotana Modern Area Rug, Contemporary Blue/Gray/Ivory 5'2\"x7'4\" [SEP] $44.99 to $33.98 [SEP] B079V5YD92 [SEP] Furniture HotSpot \u2013 Bakers Rack \u2013 White Stain - 30\" W x 15\" D x 64.5\" H [SEP] $389.99 [SEP] B09B77K9YQ [SEP] Rhomtree 36\u201d Wood Console Table Sideboard Hall Table with 2 Drawers, 1 Cabinet and 1 Shelf Modern Sofa Table for Hallway, Entryway, Living Room, Kitchen (Green with Brown Top) [SEP] $309.9"
            },
            {
                "from": "gpt",
                "rewards": 0.0,
                "loss": true,
                "value": "{\"Action\":\"click[Next >]\"}"
            },
            {
                "from": "human",
                "rewards": 0.0,
                "loss": true,
                "value": "WebShop [SEP] Instruction: [SEP] Find me space saving, easy clean baskets and storage for living room with color: grey |  beige, and price lower than 40.00 dollars [SEP] SearchInstruction: [SEP] Find me space saving, easy clean baskets and storage for living room with color: grey |  beige, and price lower than 40.00 dollars [SEP] Back to Search [SEP] Page 2 (Total results: 50) [SEP] < Prev [SEP] Next > [SEP] B06XFZXXTC [SEP] Coralpearl Inflatable Hanger Travel X 4, White Round Shoulder, Portable Folding Clothes Drying Rack in Metal Hook,Space Saving Coat Storage Set Non Slip Foldable for Home Car Camping Indoor Outdoor [SEP] $17.99 [SEP] B079N3VLRJ [SEP] Baja Night Stand / 2 Drawer / Solid Wood / Rustic Bedside Table for Bedroom, Living Room, Sofa Couch, Hall / Metal Drawer Pulls, Barnwood Finish [SEP] $237.0 [SEP] B08XX6QRK4 [SEP] Full Loft Bed with Desk and Storage Shelves, Wood Loft Bed with Ladder, for Kids, Teens, Adults (Espresso) [SEP] $549.99 [SEP] B07XTK3P89 [SEP] Lunarable Oriental Storage Toy Bag Chair, Eastern Bohem Detail Ottoman Tile Work Inspired Flowers, Stuffed Animal Organizer Washable Bag, Small Size, Turquoise Marigold [SEP] $39.99 [SEP] B09LHM4WQS [SEP] Lift Top Coffee Tables for Living Room Coffee Table with Storage, Lift Cable Coffee Table, Dining Table with Hidden Storage Compartment Chestnut [SEP] $82.99 [SEP] B09C8FZ9YS [SEP] ZZF Bookshelf Sling Bookshelf Small Volume 3 Layers Desktop Bookcase Double Drawer Storage Box Creative Preservative Wood,4 Colors, 2 Styles (Color : A-a, Size : 60X17X48CM) [SEP] $204.72 [SEP] B082KBN23S [SEP] WYSTAO Decorative Mirror Modern Minimalist Living Room Metal Mirror Wall Hanging Hall Pendant Iron Golden Soft Decoration 40/50/60/70cm (Size : 70x70cm/27.6x27.6in) [SEP] $100.0 [SEP] B08P8LRFZ4 [SEP] MTFY Cute Faux Fur Desk Chair for Girls Women, Comfy Fluffy Modern Home Office Chair with Wheels, Elegant Swivel Fuzzy Vanity Chair Makeup Arm Chair for Living Room, Bedroom [SEP] $95.99 [SEP] B09F6SCW47 [SEP] hongxinq Floating TV Stand with Storage Shelves, Chipboard Wall Mounted Hanging Television Cabinet, Modern Storage Cabinet for Living Room, Bedroom, 12Inx11.8Inx35.4In, White [SEP] $90.79 [SEP] B09MNNBDTJ [SEP] Woody BL-EF Electric Fireplace 77\" TV Stand (White/Gray) [SEP] $100.0"
            },
            {
                "from": "gpt",
                "rewards": 0.0,
                "loss": true,
                "value": "{\"Action\":\"click[Next >]\"}"
            },
            {
                "from": "human",
                "rewards": 0.0,
                "loss": true,
                "value": "WebShop [SEP] Instruction: [SEP] Find me space saving, easy clean baskets and storage for living room with color: grey |  beige, and price lower than 40.00 dollars [SEP] SearchInstruction: [SEP] Find me space saving, easy clean baskets and storage for living room with color: grey |  beige, and price lower than 40.00 dollars [SEP] Back to Search [SEP] Page 3 (Total results: 50) [SEP] < Prev [SEP] Next > [SEP] B09HXDPKJF [SEP] GXP Solid Reclaimed Wood Bedside Cabinet w/3 Drawers Nightstand Side Table [SEP] $100.0 [SEP] B09BYX42DW [SEP] Grommet Blackout Curtains for Bedroom and Living Room - 2 Panels Set Thermal Insulated Room Darkening Curtains (52 x 84 Inch, Navy Blue, 2 Panels) [SEP] $28.99 [SEP] B08SHNQJBW [SEP] Compact Versatile Design X-Shaped Kitchen Baker's Rack Microwave Stand Utility Shelf with 6 Hooks Sturdy Durable Steel, Adjustable Foot Pad [SEP] $208.99 [SEP] B09NRF2QGD [SEP] DIANDIAN Sturdy Coat Rack Stand Hall Solid Wood Coat Tree with 9 Round Hook, for Clothing Hats, Hallway Entryway Office, Easy Assembly (Color : Beige) [SEP] $311.5 [SEP] B009EEVDSQ [SEP] Queen Size Tan Traditional Japanese Floor Futon Mattresses, Foldable Cushion Mats, Yoga, Meditaion 60\" Wide X 80\" Long [SEP] $149.0 [SEP] B08L2ZDWN2 [SEP] PAVILIA Decorative Sherpa Throw Pillow Covers, Set of 2, 18x18, Light Pink Blush Fluffy Pillow Cases for Couch, Bed, Sofa|Soft Accent Cushion Cover, Shaggy Living Room Decor [SEP] $13.99 [SEP] B0924PKLQ4 [SEP] V-MORO HomePod Mini Wall Mount Holder, Outlet Mount Stand Hidden Cable Management for Apple HomePod Mini Smart Speaker Shelf Without Messy Wires Excellent Space Saving Punch-Free 2-Pack White [SEP] $19.99 [SEP] B09M7M1YB3 [SEP] Refillable Container, Easy To Clean Cream Bottle Wide Mouth Design for Easy Storage Of Specimens for Toilets [SEP] $15.5 [SEP] B09CQ45ZRB [SEP] Merry Christmas Kitchen Mat 2 PCS Cushioned Anti Fatigue Kitchen Runner Rugs Truck with Xmas Tree Red and Black Buffalo Plaid Comfort Standing Desk Mat for Office Floor Mat 19.7\"x31.5\"+19.7\"x47.2\" [SEP] $15.9 [SEP] B07DRFDWVT [SEP] JINCHAN Tie Up Valance Moroccan Tile Print Curtain Valance Bedroom Curtain Quatrefoil Flax Linen Blend Textured Geometry Lattice Rod Pocket Window Treatment Set Living Room 1 Panel 18\" L Grey [SEP] $12.99"
            },
            {
                "from": "gpt",
                "rewards": 0.0,
                "loss": true,
                "value": "{\"Action\":\"click[Next >]\"}"
            },
            {
                "from": "human",
                "rewards": 0.0,
                "loss": true,
                "value": "WebShop [SEP] Instruction: [SEP] Find me space saving, easy clean baskets and storage for living room with color: grey |  beige, and price lower than 40.00 dollars [SEP] SearchInstruction: [SEP] Find me space saving, easy clean baskets and storage for living room with color: grey |  beige, and price lower than 40.00 dollars [SEP] Back to Search [SEP] Page 4 (Total results: 50) [SEP] < Prev [SEP] Next > [SEP] B08CV7M1HV [SEP] C-DECOR Uniqueness Style Middle Coffee Table Wooden and Tempered Glass, Cocktail Table, Sofa Table for Living Room, Walnut Finish [SEP] $1200.0 [SEP] B07Q87P8DQ [SEP] Permo Vintage Rustic Industrial 3-Lights Kitchen Island Chandelier Triple 3 Heads Pendant Hanging Ceiling Lighting Fixture with Oval Cone Clear Glass Shade (Antique) [SEP] $94.99 [SEP] B08PYQVNBQ [SEP] YITAHOME 5 Tier Bookshelf, Open Freestanding 5 Shelf Bookcase 5 Tier Storage Shelf for Display and Collection, Industrial Decorative Shelf for Living Room, Home, Office, Retro Brown Bookshelf [SEP] $159.99 [SEP] B09M6VXD6W [SEP] LJP Nightstand Iron Mesh Nightstand, 3 Layer Portable Narrow Side Table can Bearing 50Kg, Sofa End Table Can Be Used as a Small Bookshelf Bedside Table (Color : Black) [SEP] $179.82 [SEP] B08HN55NGW [SEP] Prepac Entryway Shoe Cubby Console, 60\", Drifted Gray [SEP] $280.99 [SEP] B09LCM3NKN [SEP] Sheer Window Curtain Panels Cute Snowman Red Scarf Winter Snowflake,Semi Sheer Window Covering Privacy Drapes 2 Panel Sets Farm Vintage Board Green Grid Window Treatment for Kitchen/Living Room [SEP] $44.05 [SEP] B09PTQFBNT [SEP] Mulyyds Computer Desk with Drawers and Open Shelves, Modern Writing Desk Student Study Table Gaming Workstations PC Laptop Corner Desk for Small Spaces Home Office Dormitory (White, 43x19x27in) [SEP] $159.99 [SEP] B09HPMWHM4 [SEP] Generic Floss Pick, Floss Stick, Clean Flossers High Toughness with Portable Cases Hotel Travel - Blue 100Pcs [SEP] $10.99 [SEP] B09DG92LJL [SEP] HSJWOSA Decorative Rustic Entryway Console Table, 60\" Long Sofa Table with Two Different Size Drawers and Bottom Shelf for Storage Ornamental (Color : Black) [SEP] $557.72 [SEP] B09P65MG4G [SEP] Rustic Luxe Large Wooden Sofa Table, Gray [SEP] $134.0"
            },
            {
                "from": "gpt",
                "rewards": 0.0,
                "loss": true,
                "value": "{\"Action\":\"click[< Prev]\"}"
            },
            {
                "from": "human",
                "rewards": 0.0,
                "loss": true,
                "value": "WebShop [SEP] Instruction: [SEP] Find me space saving, easy clean baskets and storage for living room with color: grey |  beige, and price lower than 40.00 dollars [SEP] SearchInstruction: [SEP] Find me space saving, easy clean baskets and storage for living room with color: grey |  beige, and price lower than 40.00 dollars [SEP] Back to Search [SEP] Page 3 (Total results: 50) [SEP] < Prev [SEP] Next > [SEP] B09HXDPKJF [SEP] GXP Solid Reclaimed Wood Bedside Cabinet w/3 Drawers Nightstand Side Table [SEP] $100.0 [SEP] B09BYX42DW [SEP] Grommet Blackout Curtains for Bedroom and Living Room - 2 Panels Set Thermal Insulated Room Darkening Curtains (52 x 84 Inch, Navy Blue, 2 Panels) [SEP] $28.99 [SEP] B08SHNQJBW [SEP] Compact Versatile Design X-Shaped Kitchen Baker's Rack Microwave Stand Utility Shelf with 6 Hooks Sturdy Durable Steel, Adjustable Foot Pad [SEP] $208.99 [SEP] B09NRF2QGD [SEP] DIANDIAN Sturdy Coat Rack Stand Hall Solid Wood Coat Tree with 9 Round Hook, for Clothing Hats, Hallway Entryway Office, Easy Assembly (Color : Beige) [SEP] $311.5 [SEP] B009EEVDSQ [SEP] Queen Size Tan Traditional Japanese Floor Futon Mattresses, Foldable Cushion Mats, Yoga, Meditaion 60\" Wide X 80\" Long [SEP] $149.0 [SEP] B08L2ZDWN2 [SEP] PAVILIA Decorative Sherpa Throw Pillow Covers, Set of 2, 18x18, Light Pink Blush Fluffy Pillow Cases for Couch, Bed, Sofa|Soft Accent Cushion Cover, Shaggy Living Room Decor [SEP] $13.99 [SEP] B0924PKLQ4 [SEP] V-MORO HomePod Mini Wall Mount Holder, Outlet Mount Stand Hidden Cable Management for Apple HomePod Mini Smart Speaker Shelf Without Messy Wires Excellent Space Saving Punch-Free 2-Pack White [SEP] $19.99 [SEP] B09M7M1YB3 [SEP] Refillable Container, Easy To Clean Cream Bottle Wide Mouth Design for Easy Storage Of Specimens for Toilets [SEP] $15.5 [SEP] B09CQ45ZRB [SEP] Merry Christmas Kitchen Mat 2 PCS Cushioned Anti Fatigue Kitchen Runner Rugs Truck with Xmas Tree Red and Black Buffalo Plaid Comfort Standing Desk Mat for Office Floor Mat 19.7\"x31.5\"+19.7\"x47.2\" [SEP] $15.9 [SEP] B07DRFDWVT [SEP] JINCHAN Tie Up Valance Moroccan Tile Print Curtain Valance Bedroom Curtain Quatrefoil Flax Linen Blend Textured Geometry Lattice Rod Pocket Window Treatment Set Living Room 1 Panel 18\" L Grey [SEP] $12.99"
            },
            {
                "from": "gpt",
                "rewards": 0.0,
                "loss": true,
                "value": "{\"Action\":\"click[B07DRFDWVT]\"}"
            },
            {
                "from": "human",
                "rewards": 0.0,
                "loss": true,
                "value": "WebShop [SEP] Instruction: [SEP] Find me space saving, easy clean baskets and storage for living room with color: grey |  beige, and price lower than 40.00 dollars [SEP] SearchInstruction: [SEP] Find me space saving, easy clean baskets and storage for living room with color: grey |  beige, and price lower than 40.00 dollars [SEP] Back to Search [SEP] < Prev [SEP] color [SEP] *soft grey [SEP] soft grey [SEP] taupe [SEP] JINCHAN Tie Up Valance Moroccan Tile Print Curtain Valance Bedroom Curtain Quatrefoil Flax Linen Blend Textured Geometry Lattice Rod Pocket Window Treatment Set Living Room 1 Panel 18\" L Grey [SEP] Price: $12.99 [SEP] Rating: N.A. [SEP] Description [SEP] Features [SEP] Reviews [SEP] Buy Now"
            },
            {
                "from": "gpt",
                "rewards": 0.2,
                "loss": true,
                "value": "{\"Action\":\"click[Buy Now]\"}"
            }
        ],
        "id": "webshop_0"
    },

Log: 
```
PASTE THE LOGS HERE.
```

Screenshots:
You may attach screenshots if it better explains the issue.

**Comment by Tree-Shu-Zhao** [comment]
Hi! Have you figured out the tuning process? I have multimodal/text mixed samples. After fine-tuning, I found the performance on the pure textual data samples significantly dropped. May I ask if you got similar results, or could you provide any suggestions about how to prevent the performance drop?
